cmd/ocsp-updater/main.go

```diff
@@ -129,11 +140,63 @@ func newUpdater(
 		})
 	}
 
-	updater.ocspMinTimeToExpiry = config.OCSPMinTimeToExpiry.Duration
+	if config.AkamaiBaseURL != "" {
```

Comment:
(Your response might be "but they might not run a CDN because it's a private CA" and I'm like "why are they running an ocsp-responder and ocsp-updater at all, then?")